src/main/java/reposense/parser/CsvParser.java

```diff
@@ -76,6 +77,23 @@ public CsvParser(Path csvFilePath) throws IOException {
         return results;
     }
 
+    /**
+     * Returns an array of Strings representing the header of a CSV file.
+     *
+     * @param reader A BufferedReader opened on the CSV file.
+     * @return an array of Strings where each Strings is a column name from the CSV header.
+     * @throws IOException
+     */
+    private String[] getHeader(BufferedReader reader) throws IOException {
+        String currentLine = "";
+
+        while (currentLine.length() == 0) {
```

Comment:
May need to test this on an empty file.

Reply:
Thanks. I have added a test case for this in `CsvParserTest.java`